.github/workflows/imaginary-update.yml

```diff
@@ -0,0 +1,36 @@
+# Inspired by https://github.com/nextcloud/docker/blob/master/.github/workflows/update-sh.yml
+name: imaginary-update
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron:  '00 12 * * *'
+
+jobs:
+  run_update:
+    name: update to latest imaginary commit on master branch
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Run imaginary-update
+      run: |
+        # Inspired by https://github.com/nextcloud/docker/blob/master/update.sh
+
+        # Imaginary
+        imaginary_version="$(
+          git ls-remote https://github.com/h2non/imaginary master \
+            | cut -f1 \
+            | tail -1
```

Comment:
```suggestion
            | tail -1 \
```

Reply:
why? the other workflos also dont have a backslash at the end whenbusingcthis syntax

Reply:
Right